adhoc_reports/parseData.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pandas as pd
+from pprint import pprint
+
+os.system("wget --no-check-certificate \'https://spreadsheets.google.com/spreadsheet/ccc?key=1UHyMs5Havel4iEBMPVx7ZY4HMPuhRilIw59a5g733p8&output=xlsx\' -O data.xlsx")
```

Comment:
Exposing your credentials by explicitly hard coding it is a bad idea. The key=1UH**3p8 part should be hidden in a config file that should not be hosted on github; it should reside at the amazon instance node. Or at least passed as a command line argument.